src/components/reducer/CounterReducer.jsx

```diff
@@ -0,0 +1,17 @@
+import { counter } from "../../utilities";
+
+const ReducerCounter = (state = counter, action) => {
+  switch (action.type) {
+    case "counterIncrease":
+      return { count: state.count + 1 };
+    case "counterDecrease":
+      if (state.count < 1) {
+        return state;
+      }
+      return { count: state.count - 1 };
+    default:
+      return state;
+  }
+};
+
```

Comment:
create a redux folder inside src and move all redux related files and folder inside the redux folder